build.gradle.kts

```diff
@@ -118,14 +118,21 @@ rootDir.walk().maxDepth(4).filter { it.isFile && it.extension == "Dockerfile" }.
             group = "Docker"
             description = "Builds the $name worker Docker image."
 
+            val imageName = if (name == "Base") {
+                "ort-server-base-image:latest"
+            } else {
+                dependsOn("buildBaseWorkerImage")
```

Comment:
Nit: Probably also use an "absolute" task name here, with a leading `:`, like in line 157 below.